packages/suite/src/support/messages.ts

```diff
@@ -340,6 +340,34 @@ export default defineMessages({
         id: 'TR_COINMARKET_SWAP_DEX_MODAL_TERMS_6',
         dynamic: true,
     },
+    TR_ALL_NETWORKS: {
+        defaultMessage: 'All networks ({networkCount})',
+        id: 'TR_ALL_NETWORKS',
+    },
+    TR_SELECT_TOKEN: {
+        defaultMessage: 'Select a token',
+        id: 'TR_SELECT_TOKEN',
+    },
+    TR_SELECT_NAME_OR_ADDRESS: {
+        defaultMessage: 'Search by name or paste address',
```

Comment:
`Search by name, symbol...` I am not sure about address. There are also network tokens (bitcoin, ethereum, solana, waves, litecoin,...)

Reply:
In such cases you should ideally know that you can search just by name.

Reply:
Search by name, symbol, network or contract address